packages/blockchain-environment-v1/src/FabricEnvironmentConnection.ts

```diff
@@ -432,8 +432,17 @@ export class FabricEnvironmentConnection implements IFabricEnvironmentConnection
 
     private async setNodeContext(nodeName: string): Promise<void> {
         const node: FabricNode = this.getNode(nodeName);
-        const fabricWallet: IFabricWallet = await this.getWallet(nodeName);
-        await fabricWallet['setUserContext'](this.client, node.identity);
+        const fabricWallet: FabricWallet = await this.getWallet(nodeName);
+        const fabricIdentity: FabricIdentity = await fabricWallet.getIdentity(node.identity);
+        const identity: any = {
```

Comment:
Do we re-use this anywhere - if so, is it worth creating an interface?

Reply:
that whole function is going away so not worth it